gh-132671: Do not match host+port with proxy bypass macOS rules, match host only. (#132672)
python/cpython: Misc/NEWS.d/next/macOS/2025-04-18-06-36-46.gh-issue-132671.FRLb6i.rst

> Fix issue where system proxy bypass configuration doesn't work with URL with port part.

Reviewer: - We need some context about what module/function is actually being fixed.
- "system proxy bypass configuration doesn't work" is a bit wordy.
```suggestion
Fix proxy bypass on URLs with a port in :mod:`urllib.request`.
```
